src/content/blog.njk

```diff
@@ -1,33 +1,17 @@
 ---
 layout: base.njk
-pagination:
-  data: posts
-  size: 5
 ---
 
 {% block content %}
-	{% set allArticles = pagination.items %}
+<ul class="blog-articles">
+  {%- for post in posts -%}
+    {%- set postFeaturedImage = media | mediaGetMediumURL(post.featuredImage) -%}
+    {%- set postPermalink = post.permalink -%}
+    {%- set postTitle = post.title -%}
+    {%- set postDate = post.date | readableDate -%}
+    {%- set postExcerpt = post.excerpt -%}
```

Comment:
All of this is needed to be able to share a common template (include) for both the actual blog content and the WP theme 🚀